etc/profile-a-l/chromium-common.profile

```diff
@@ -40,5 +39,9 @@ disable-mnt
 ?BROWSER_DISABLE_U2F: private-dev
 # private-tmp - problems with multiple browser sessions
 
+# prevents access to passwords saved in GNOME Keyring and KWallet, also breaks Gnome connector
+# dbus-user none
+# dbus-system none
```

Comment:
Can we block the system-bus?

Reply:
@rusty-snake Not sure, I'm actually not that familiar with chrome-based browsers. So if you say the system bus can be blocked without issues, I'm ok with that. Thanks for the input, I didn't want to commit this without seeking some advice.

Reply:
Password are per user, therefore on the session-bus. 

In general: I block the system-bus in my `pre-globals.local` for around two month now, and only allowed it for d-feet and searhorse. IHMO we can add `dbus-system none` to the most profiles.

Reply:
But I'm not sure if it maybe still breaks something in chromium.

Reply:
@rusty-snake That makes sense, thanks for your input, as always. I've blocked the system bus, we can change this later if we get issue reports.